### ISSUE TITLE
program not getting past mod selection screen

### ISSUE THREAD

**Issue by SlackerNet** [EfficiencyVI/adcmmp#6]
for some reason just after 12-25-16 the program will no longer access curse downloads. You select the pack click ok then it does nothing. Any chance of an update Please

**Comment by SlackerNet** [comment]
update:   I was looking at the program and noticed the links were http in the program and https on the site. I changed them hoping it would work. it was a fail the program still doesn't do anything after selecting the modpack and hitting ok  button : /
note: i dont know much about programming i was just trying stuff out to see if it worked

**Comment by AarynSmith** [comment]
Looks like curse is redirecting to a 404 page for modpacks now. Removing the --quiet option from the wget's yeilds this:
```
--2017-01-10 19:42:34--  http://minecraft.curseforge.com/modpacks/256183-modpack
Resolving minecraft.curseforge.com (minecraft.curseforge.com)... 104.16.82.211, 104.16.83.211, 104.16.85.211, ...
Connecting to minecraft.curseforge.com (minecraft.curseforge.com)|104.16.82.211|:80... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://minecraft.curseforge.com/modpacks/256183-modpack [following]
--2017-01-10 19:42:34--  https://minecraft.curseforge.com/modpacks/256183-modpack
Connecting to minecraft.curseforge.com (minecraft.curseforge.com)|104.16.82.211|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2017-01-10 19:42:34 ERROR 404: Not Found.
```

**Comment by SlackerNet** [comment]
i wonder how they tell the difference for the redirect....example you downloading the files manually vs the program downloading the files. IMO curse is nothing but a bunch of fkn linux haters

**Comment by mmarquezs** [comment]
@SlackerNet  That's not true, they simply changed some urls, the urls don't work manually either. 

They joined all the urls in one, modpacks, mods etc... are all at https://www.curseforge.com/projects/<projectid>

It's simple to fix. I just did a pull request with the code fixed https://github.com/EfficiencyVI/adcmmp/pull/7

Meanwhile you can use my fork:
> wget -qO - https://cdn.rawgit.com/mmarquezs/adcmmp/ddc6d4c7/curse.bash | bash -

**Comment by SlackerNet** [comment]
sorry i have been offline. I will try your ver out. thank you for the link,  this has been frustrating to say the least
